src/components/update-dialog/UpdateDialog.tsx

```
@@ -0,0 +1,267 @@
+      <p className="text-muted-foreground py-4 text-center text-sm">
+        No release notes for this version.
+      </p>
+    )
+  }
+
+  const html = marked.parse(notes, { async: false })
+
+  return (
+    <div
+      className="release-notes max-h-[300px] overflow-y-auto rounded-md border p-4"
+      dangerouslySetInnerHTML={{ __html: html }}
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)